test.sh

```diff
@@ -78,7 +77,7 @@ python -m pytest -n auto .
 cd ..
 
 # Build Sphinx docs.
-pip install -r requirements/requirements-docs.txt
+pip install -e ."[docs]"
```

Comment:
```suggestion
pip install -e ".[docs]"
```

Reply:
Sounds good, Addressed in [46d8a48](https://github.com/deepmind/optax/pull/513/commits/46d8a48f87403bed9b7fb58ad3dd79e1ed4b18c8)